src/Microsoft.Graph.Auth/Microsoft.Graph.Auth.csproj

```diff
@@ -0,0 +1,45 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Description>Microsoft Graph Auth Library implements auth functionality used by Microsoft Graph Client Libraries.</Description>
+    <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
+    <AssemblyTitle>Microsoft Graph Auth Library</AssemblyTitle>
+    <VersionPrefix>0.1.0</VersionPrefix>
+    <FileVersion>0.1.0</FileVersion>
+    <AssemblyVersion>0.1.0</AssemblyVersion>
+    <Authors>Microsoft</Authors>
+    <TargetFrameworks>netstandard1.3;net45;</TargetFrameworks>
+    <AssemblyName>Microsoft.Graph.Auth</AssemblyName>
+    <PackageId>Microsoft.Graph.Auth</PackageId>
+    <PackageTags>Microsoft Office365;Graph;GraphServiceClient;Outlook;OneDrive;AzureAD;GraphAPI;Productivity;SharePoint;Intune;SDK</PackageTags>
+    <PackageReleaseNotes>
+    </PackageReleaseNotes>
+    <PackageProjectUrl>https://developer.microsoft.com/graph</PackageProjectUrl>
+    <PackageLicenseUrl>http://aka.ms/devservicesagreement</PackageLicenseUrl>
+    <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
+    <GenerateAssemblyTitleAttribute>false</GenerateAssemblyTitleAttribute>
+    <GenerateAssemblyDescriptionAttribute>false</GenerateAssemblyDescriptionAttribute>
+    <GenerateAssemblyConfigurationAttribute>false</GenerateAssemblyConfigurationAttribute>
+    <GenerateAssemblyCompanyAttribute>false</GenerateAssemblyCompanyAttribute>
+    <GenerateAssemblyProductAttribute>false</GenerateAssemblyProductAttribute>
+    <GenerateAssemblyCopyrightAttribute>false</GenerateAssemblyCopyrightAttribute>
+    <GenerateNeutralResourcesLanguageAttribute>false</GenerateNeutralResourcesLanguageAttribute>
+    <GenerateAssemblyVersionAttribute>false</GenerateAssemblyVersionAttribute>
+    <GenerateAssemblyFileVersionAttribute>false</GenerateAssemblyFileVersionAttribute>
+    <SignAssembly>false</SignAssembly>
+    <DelaySign>true</DelaySign>
+    <!--Skip 1.13.0-->
+    <Version>0.1.0</Version>
```

Comment:
Change Version to VersionPrefix. When we have Version = VersionPrefix + VersionSuffix.